bug: Error: Unknown issue id "AndroidGradlePluginVersion"

**Describe the bug**
Android build error:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':capacitor-android:lint'.
> Lint found errors in the project; aborting build.
  
  Fix the issues identified by lint, or add the following to your build script to proceed with errors:
  ...
  android {
      lintOptions {
          abortOnError false
      }
  }
  ...
  Errors found:
  
  /home/runner/work/capacitor-plugin-demo/capacitor-plugin-demo/node_modules/@capacitor/android/capacitor: Error: Unknown issue id "AndroidGradlePluginVersion", found in /home/runner/work/capacitor-plugin-demo/capacitor-plugin-demo/node_modules/@capacitor/android/capacitor/lint.xml [LintError]
```

**To Reproduce**
See https://github.com/robingenz/capacitor-plugin-demo/runs/2632018092

**Expected behavior**
No error.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -4,10 +4,10 @@ buildscript {
 
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
+        classpath 'com.android.tools.build:gradle:4.2.0'
         classpath 'com.google.gms:google-services:4.3.3'
         classpath 'com.google.firebase:perf-plugin:1.3.5' // Firebase Performance Monitoring plugin
         classpath 'com.google.firebase:firebase-crashlytics-gradle:2.5.2' // Firebase Crashlytics plugin
@@ -22,7 +22,7 @@ apply from: "variables.gradle"
 allprojects {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
 }
 

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.1'
+        classpath 'com.android.tools.build:gradle:4.1.3'
         classpath 'com.google.gms:google-services:4.3.3'
         classpath 'com.google.firebase:perf-plugin:1.3.5' // Firebase Performance Monitoring plugin
         classpath 'com.google.firebase:firebase-crashlytics-gradle:2.5.2' // Firebase Crashlytics plugin

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -4,7 +4,7 @@ buildscript {
 
     repositories {
         google()
-        mavenCentral()
+        jcenter()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:4.2.0'
@@ -22,7 +22,7 @@ apply from: "variables.gradle"
 allprojects {
     repositories {
         google()
-        mavenCentral()
+        jcenter()
     }
 }
 

--- a/android/gradle/wrapper/gradle-wrapper.properties
+++ b/android/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
-#Sat Mar 06 17:29:54 CET 2021
+#Thu May 20 19:26:57 CEST 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip

